index.js

```diff
@@ -157,7 +189,11 @@
       }
     };
     return signer;
-  };
+  }
+
+  const onUsbReady = async (callback) => {
```

Comment:
Does the terminal version need a check in onUsbReady before calling the callback?

This is one thing I haven't tested yet.

Reply:
I've also never tested this package outside of a browser; does this package even support non-browser HW connectivity?